starfish/experiment/builder/image.py

```diff
@@ -0,0 +1,67 @@
+from io import BytesIO
```

Comment:
This produces `starfish.image` and `starfish.experiment.builder.image`. I wonder if there is a more intuitive construction. I think you have the stuff in the right place but perhaps `image` is not the right name for this module. 

I feel like `io` is a better descriptor, perhaps something along this line that doesn't conflict with the stdlib?

Reply:
Renamed to imagedata.